source/as-stream.ts

```diff
@@ -1,144 +0,0 @@
-import duplexer3 = require('duplexer3');
```

Comment:
Got no longer uses EventEmitter to emit the request and the response. It is based directly on Duplex streams.